<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="6" passed="3" failed="3" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2024-11-09T12:32:35 IST" name="Suite" finished-at="2024-11-09T12:33:15 IST" duration-ms="39545">
    <groups>
      <group name="smoke">
        <method signature="TC001_LoginPageTest.verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" name="verifyLOgin" class="testCases.TC001_LoginPageTest"/>
      </group> <!-- smoke -->
      <group name="sanity">
        <method signature="TC001_LoginPageTest.verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" name="verifyLOgin" class="testCases.TC001_LoginPageTest"/>
      </group> <!-- sanity -->
    </groups>
    <test started-at="2024-11-09T12:32:35 IST" name="Test" finished-at="2024-11-09T12:33:15 IST" duration-ms="39545">
      <class name="testCases.TC001_LoginPageTest">
        <test-method is-config="true" signature="setup(java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:32:36 IST" name="setup" finished-at="2024-11-09T12:32:45 IST" duration-ms="8709" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[chrome]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setup -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:32:50 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:32:56 IST" duration-ms="6013" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Inpatient Ward]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PASS]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:32:56 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:32:59 IST" duration-ms="2841" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Isolation Ward]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[FAIL]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error During Login: Invalid username/password. Please try again.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:32:59 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:33:04 IST" duration-ms="4790" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Laboratory]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PASS]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:33:04 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:33:09 IST" duration-ms="5053" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Outpatient Clinic]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[PASS]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:33:09 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:33:11 IST" duration-ms="2673" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Adn123]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Pharmacy]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[FAIL]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error During Login: Invalid username/password. Please try again.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method signature="verifyLOgin(java.lang.String,java.lang.String,java.lang.String,java.lang.String)[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:33:12 IST" name="verifyLOgin" data-provider="LoginCredentials" finished-at="2024-11-09T12:33:14 IST" duration-ms="2149" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[admin]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[Admin128]]>
              </value>
            </param>
            <param index="2">
              <value>
                <![CDATA[Registration Desk]]>
              </value>
            </param>
            <param index="3">
              <value>
                <![CDATA[FAIL]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Error During Login: Invalid username/password. Please try again.]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Error During Login: Invalid username/password. Please try again.
at org.testng.Assert.fail(Assert.java:111)
at testCases.TC001_LoginPageTest.verifyLOgin(TC001_LoginPageTest.java:57)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
at java.base/java.lang.reflect.Method.invoke(Method.java:580)
at org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:141)
at org.testng.internal.invokers.TestInvoker.invokeMethod(TestInvoker.java:686)
at org.testng.internal.invokers.TestInvoker.invokeTestMethod(TestInvoker.java:230)
at org.testng.internal.invokers.MethodRunner.runInSequence(MethodRunner.java:63)
at org.testng.internal.invokers.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:992)
at org.testng.internal.invokers.TestInvoker.invokeTestMethods(TestInvoker.java:203)
at org.testng.internal.invokers.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:154)
at org.testng.internal.invokers.TestMethodWorker.run(TestMethodWorker.java:134)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
at org.testng.TestRunner.privateRun(TestRunner.java:739)
at org.testng.TestRunner.run(TestRunner.java:614)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:421)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:413)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:373)
at org.testng.SuiteRunner.run(SuiteRunner.java:312)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:95)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1274)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1208)
at org.testng.TestNG.runSuites(TestNG.java:1112)
at org.testng.TestNG.run(TestNG.java:1079)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:293)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:91)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- verifyLOgin -->
        <test-method is-config="true" signature="teaeDown()[pri:0, instance:testCases.TC001_LoginPageTest@52851b44]" started-at="2024-11-09T12:33:14 IST" name="teaeDown" finished-at="2024-11-09T12:33:15 IST" duration-ms="633" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- teaeDown -->
      </class> <!-- testCases.TC001_LoginPageTest -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
